src/plugin-hooks/usePagination.js

```diff
@@ -52,13 +52,14 @@ function useMain(instance) {
 
   useLayoutEffect(
     () => {
```

Comment:
If this wasn't working before then I think the bug is deeper in some memoization.